TextRPG_Team25/Battle.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Numerics;
+using System.Threading;
+using TextRPG_Team25.Core;
+using TextRPG_Team25.UI;
+
+namespace TextRPG_Team25
+{
+    internal class Battle
+    {
+        public static void BattleResult()
+        {
+           if (isVictory)
+           {
+                Utils.ColoredText("Battle! -Result-", ConsoleColor.DarkYellow);
+                Utils.ColoredText("Victory!", ConsoleColor.Green);
+                Console.WriteLine("던전에서 몬스터 ");
+                Utils.ColoredText($"{monster.Count}", ConsoleColor.Red);
```

Comment:
정우님이 작성하신 이름에 맞게 교체해놨습니당
```suggestion
                Utils.ColoredText($"{FieldMonster.Count}", ConsoleColor.Red);
```